src/Auth/LogIn/LogIn.test.tsx

```diff
@@ -12,8 +12,22 @@ jest.mock('react-firebase-hooks/auth')
 const mockSignIn = mocked(useSignInWithEmailAndPassword, true)
 
 const Dashboard = () => <div>I am the dashboard</div>
+const CreateAccount = 'If you do not have an account, click here</a> to contact us and create it'
 
 describe(LogIn, () => {
+  it('It allows for user creation', async () => {
```

Comment:
```suggestion
  it('allows for user creation', async () => {
```

no need to write it twice :)